docs/Basics/index.md

```diff
@@ -6,37 +6,38 @@ The variety of AEC (Architecture, Engineering and Construction) commercial softw
 
-This is where the BHoM comes in. Thanks to the central common language, it is possible to interoperate between many different software. Instead of creating translators between every possible pair of software, you just have to write one single translator between BHoM and a target software, to then connect to all the others software. 
-We call those translators **Adapter**s.
+This is where the BHoM comes in. Thanks to the central common language, it is possible to interoperate between many different programs. Instead of creating translators between every possible pair of applications, you just have to write one single translator between BHoM and a target software, to then connect to all the others.
+
+We call those translators **Adapters**.
 
 ![](https://raw.githubusercontent.com/BHoM/documentation/main/Images/InteropA.png)
 
 
 
 ## Linking functionality
-Most often, AEC industry experts creates ad-hoc functions and tools when the need arises. Common examples are large, complex spreadsheets, VBA macros for Excel, but also small and large User Interface programs written in C#, Python or other means.
+Most often, AEC industry experts creates ad-hoc functions and tools when the need arises. Common examples are large, complex spreadsheets, VBA macros for Excel, but also small and large User Interface programs written in C#, Python or languages.
 
-In such a large sector, most problems you encounter are likely to have been solved before by someone in your organisation, or outside. What frequently happens is that the wheel gets reinvented. Additionally, when a script is created, it often exist in isolation, and is used only by a small group of people. 
+In such a large sector, most problems you encounter are likely to have been solved before by someone in your organisation, or outside. What frequently happens is that the wheel gets reinvented. Additionally, when a script is created, it often exists in isolation, and is used only by a small group of people. 
 
 Let's take the example of a user wanting to create some functionality in an Excel macro. This has several issues:
 
 - **Maintanability issues**. Frequently, only the original creator of a Macro knows how to develop or maintain it. If they are not around and a problem arises, the Macro is often just thrown away.
 
-- **Shareability issues**. For example, if the creator of a Macro that does some function in a spreadsheet wants to share this functionality with one of their clients, they end up sharing the spreadsheet with the Macro in it, effectively ending up sharing the source code, fruit of their hard work, which could end up being copied.
+- **Shareability issues**. For example, if the creator of a Macro that performs some function in a spreadsheet wants to share this functionality with one of their clients, they often end up sharing the spreadsheet with the Macro in it, effectively sharing the source code and the fruit of their hard work, which ends up just being copied.
```

Comment:
I disagree with the changes made here. The first tries to limit how the sharing occurs (adding the word `often`), while the latter suggests the work will always be copied.

I agree the wording around `the fruit of their hard work` on the original isn't great either, but the `fruit of their hard work` is not separate from the `source code` - they are the same thing. Thus, I'm suggesting using brackets rather than an `and` which suggests they're separate items.

```suggestion
- **Shareability issues**. For example, if the creator of a Macro that performs some function in a spreadsheet wants to share this functionality with one of their clients, they end up sharing the spreadsheet with the Macro in it, effectively sharing the source code (the fruit of their hard work), which could end up being copied.
```